src/command.rs

```diff
@@ -102,11 +102,13 @@ pub enum Command {
     Shift(ShiftMode, Option<i32>),
     Search(String),
     Help,
+    Noop,
 }
 
 impl fmt::Display for Command {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let repr = match self {
+            Command::Noop => "no operation".to_string(),
```

Comment:
This representation should resolve to the command name instead

Reply:
resolved and rebased into d33147c